Formula/go@1.15.rb

```diff
@@ -50,6 +50,9 @@ def install
     bin.install_symlink Dir[libexec/"bin/go*"]
 
     system bin/"go", "install", "-race", "std"
+
+    # Binaries built for an incompatible architecture
+    rm_rf Dir[libexec/"src/runtime/pprof/testdata"]
```

Comment:
```suggestion
    (libexec/"src/runtime/pprof/testdata").rmtree
```
This guarantees we error out when this directory no longer exists.

Reply:
Can you make that edit in the unversioned formula too?